service/frontend/fx.go

```diff
@@ -267,6 +269,7 @@ func GrpcServerOptionsProvider(
 		namespaceLogInterceptor.Intercept, // TODO: Deprecate this with a outer custom interceptor
 		metrics.NewServerMetricsContextInjectorInterceptor(),
 		authInterceptor.Intercept,
+		namespaceHandoverInterceptor.Intercept,
```

Comment:
Can you add some comments explaining why it must be in this place?

Reply:
Is the handover state checking logic in StateValidationIntercept removed?

Reply:
I did not remove in this PR as I added the feature flag. Once we can confident we can remove it.

Reply:
do you want to disabled that the check in StateValidationIntercept if the feature flag is enabled?

Reply:
Yes. Once I validate the function and set default to enable I will remove it